src/deepsparse/server/monitoring.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
let's rename this file to something more specific to the functionality given monitoring also has (more relevant) connotations to our logging functionality